client/lib/media/utils/create-transient-media-id.js

```diff
@@ -12,5 +10,5 @@ const { uniqueId } = impureLodash;
  * @param {string} moreSpecificPrefix can be used to further specify the prefix
  */
 export function createTransientMediaId( moreSpecificPrefix = '' ) {
-	return uniqueId( `media-${ moreSpecificPrefix }` );
+	return `media-${ moreSpecificPrefix }${ uuid().replace( /-/g, '' ) }`;
```

Comment:
Do we need to do the `.replace` call everywhere? Seems to me we don't really care about what exactly is the returned string's content, as long as it's random enough.

Reply:
Maybe we don't. I'm adding it just to reduce friction by at least removing any non-alphanumeric characters in this migration. If you feel strongly about it I can remove it.

Reply:
Let's remove it from both PRs unless we know that the ID is interpreted somehow and the `-` characters are an obstacle. Otherwise it's just noise.

Reply:
Sounds good. Removed in https://github.com/Automattic/wp-calypso/pull/52672/commits/c1b19a514b3494212396dffeb9045076da3c566c